embedded-graphics/src/primitives/triangle/styled.rs

```diff
@@ -229,7 +229,6 @@ mod tests {
         let styled = triangle.into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 1));
 
         let mut tri_display: MockDisplay<BinaryColor> = MockDisplay::new();
-        // FIXME: Triangles should not be overdrawing
         tri_display.set_allow_overdraw(true);
```

Comment:
Is `set_allow_overdraw` still required for this test?

Reply:
Oh jeez... I deleted the comment but not the line :facepalm: